src/__tests__/Search/CompareOverTime.test.tsx

```diff
@@ -155,19 +155,19 @@ describe('Compare Over Time', () => {
       name: 'mozilla-central',
     });
     await user.click(mozRepoItem);
-    expect(mozRepoItem).toBeInTheDocument();
+    expect(baseDropdown).toHaveTextContent('mozilla-central');
```

Comment:
Another case where I believe `await user.click()` resolves later than before.

Previously the option we just clicked was still present... but I believe this was a mistake! as the dropdown disappears after a click. I replaced by checking whether the button now reads the new selected option.

(same below)